Monika After Story/game/event-handler.rpy

```diff
@@ -2416,6 +2824,22 @@ init python:
         return u_count != count
 
 
+    def mas_areEventsPaused():
```

Comment:
moved this API over into event list. I don't think we can transfer the var over because its been released. Better to avoid breaking submods. 

Also generally speaking, global vars are for things that may change often and should be accessible for general usage. this unpause dt fits those categories.